### ISSUE TITLE
Add GitHub fancy buttons to README with build status and so on

### ISSUE THREAD

**Issue by holtzmak** [holtzmak/Humane-Transport#88]
As the title says, we should consider added repo status buttons/links/icons to our README to spice it up and be more interesting to people just passing by.

**Comment by holtzmak** [comment]
We should update the README to be more informative and impressive, because we're well into development:
- [x] Should probably update README with the project description on the website - it's more informative.
- [x] What about adding documentation guide as well (no links as files change often) to explain what we used for design.
- [x] I don't think we've listed anywhere what tools our project is using: Flutter, firebase, cross-platform, etc.
